bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -0,0 +1,130 @@
+package edu.gemini.model.p1.overheads
+
+import java.time.Duration
+
+import edu.gemini.model.p1.immutable._
+
+import scalaz.Scalaz._
+
+// This is not the ideal package for this to live in, but to avoid circular bundle references, we put it here.
+sealed trait Overheads {
+  def partnerOverheadFraction: Double
+  def acquisitionOverhead: Duration
+  def typicalExpTime: Option[Duration]
```

Comment:
This may not be actually necessary, since it is usually used in the calculation (in the spreadsheet listed in the description) for `otherOverheadFraction`, I believe.